src/_blog/cryptotls.md

```diff
@@ -0,0 +1,58 @@
+
+After long discussions (both over Zoom and on the GitHub issue), we arrived at a proposal that was much cleaner than our homegrown qtls API.
+
+Adding support for 0-RTT was a fairly large endeavor. This is because the TLS and the QUIC stack need to coordinate quite a lot to enable this feature. For example, both the client and the server need to remember certain configuration parameters (called QUIC transport parameters) from the original QUIC connection. Among others, these transport parameters include values like flow control windows (i.e. how many bytes a client is allowed to send on a newly established stream) and how many streams the client is allowed to open. This information is needed for the client to stay within these limits when resuming a connection. The server typically encrypts these values and stores them in the session ticket. When the client restores the session, it sends the session ticket to the server (as part of the TLS ClientHello message), allowing the server to restore the transport parameters (without having to persist them after a connection closes!). The client also needs to store some its parameters alongside the ticket, so it can restore them when resuming the session.
+
+This means that crypto/tls needed to support adding data to session tickets, both the client and server sides. It turned out that the API required for this would also solve a large number of other longstanding [issues related to session tickets](https://github.com/golang/go/issues/60105).
+
+This is only one of the problems we had to solve to make 0-RTT work. Another complication arises from the fact that the server can reject 0-RTT for any reason (typically DoS protection or because it doesn’t consider the QUIC transport parameters acceptable anymore). In that case, 0-RTT packets are discarded and the handshake continues as a regular (1-RTT) QUIC handshake. This necessitated a new [API for deciding about 0-RTT rejection](https://github.com/golang/go/issues/60107) and for clients to be informed of it.
+
+## Current Status
+
+After an intense period of collaboration and development, we are thrilled to have all these changes included in the Go 1.21 release. The implementation of these proposals in the standard library was performed by Damien Neil and Filippo Valsorda. On the quic-go side, we made use of the new APIs as soon as the changes were published. This allowed us to use quic-go's test suite to stress-test the implementation. This was really valuable, since quic-go’s test suite is quite extensive and covers a large number of corner cases. And it indeed found a small issue, which we subsequently [fixed](https://go-review.googlesource.com/c/go/+/498215).
```

Comment:
I think you should make "Damien Niel" and "Filippo Valisorda" links to their respective GH accounts.